torchdynamo/variables/torch.py

```diff
@@ -398,3 +401,115 @@ def handle_ntuple(value):
             return variables.LambdaVariable(handle_ntuple, **options)
         else:
             return handle_ntuple(args[0])
+
+
+class TorchPyOperator(VariableTracker):
+    def __init__(self, value, **kwargs):
+        super(TorchPyOperator, self).__init__(**kwargs)
+        self.value = value
+
+    def call_function(
+        self, tx, args: "List[VariableTracker]", kwargs: "Dict[str, VariableTracker]"
+    ) -> "VariableTracker":
+        from torch.utils._python_dispatch import TorchDispatchMode
+        from torch.utils._python_dispatch import enable_torch_dispatch_mode
+
+        from . import ListVariable
+        from . import TensorVariable
+        from . import UserFunctionVariable
+
+        assert kwargs is None or len(kwargs) == 0, "kwargs are not supported, yet"
+
+        def unwrap_real(arg):
+            if isinstance(arg, TensorVariable):
+                return arg.as_proxy().node.meta["example_value"]
```

Comment:
Using example values is unsound.

Reply:
Yea. I wasn't sure what else to use here.

It felt alright because we enforce TensorVariable to have it in order to propagate:

```
def propagate_args_kwargs(node):
        def visit(n: torch.fx.Node):
            return n.meta["example_value"]
```

That being said, I'd rather not rely on it. Open to suggestions on how best to trace this!

Reply:
Discussed offline - added guard safety check.